ports/windows/msvc/sources.props

```diff
@@ -35,4 +34,16 @@
     <ClInclude Include="$(PyBaseDir)ports\windows\*.h" />
     <ClInclude Include="$(PyBaseDir)ports\windows\msvc\*.h" />
   </ItemGroup>
+  <ItemGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
```

Comment:
This can be simplified to oonly look at Platform for the condition, since the Configuration doesn't matter.

Reply:
You're right. I'll remove the Configuration.